modules/mirror-registry-localhost.adoc

```diff
@@ -0,0 +1,59 @@
+// Module included in the following assemblies:
+//
+// * installing/installing-mirroring-installation-images.adoc
+
+[id="mirror-registry-localhost{context}"]
+
+= Local host mirroring with mirror registry for {product-title}
+
+This procedure introduces how to install Red Hat Quay on a local host using the `mirror-registry` tool. By doing so, users can create a registry to hold a mirror of {product-title} images.
+
+[NOTE]
+====
+Installing Red Hat Quay using the `mirror-registry` CLI tool makes several changes to your machine. Upon installation, an `/etc/quay-install` directory is created, which contains installation files, local storage, and the config bundle. Trusted SSH keys are generated in case the deployment target is the local host, and systemd files on the host machine are set up to ensure that container runtimes are persistent. Additionally, an initial user named `init` is created with an auto-generated password. All access credentials are printed at the end of the install routine.
+====
+
+.Procedure
+
+//placeholder link
+. Download the `mirror-registry.tar.gz` package for the latest version of mirror registry found on the link:https://github.com/quay/openshift-mirror-registry/releases[mirror registry releases page].
+
+. Install Red Hat Quay on your local host with your current user account by using the `mirror-registry` tool:
++
+[source,terminal]
+----
+$ ./mirror-registry install \
+--quayHostname <host.example.com> \
+--quayRoot <example/directory/name>
+----
```

Comment:
It is better to add `sudo` in this command line like what line 51 of current page does.